Extension/src/Debugger/configurationProvider.ts

```diff
@@ -34,6 +39,30 @@ abstract class CppConfigurationProvider implements vscode.DebugConfigurationProv
             return undefined;
         }
 
+        // Modify WSL config on 64-bit VSCode for OpenDebugAD7
+        if (os.platform() === 'win32' &&
+            process.arch === 'x64' &&
```

Comment:
Why does `process.arch` mater for this code?

Reply:
I had a thought this only mattered for 64bit. I guess it would matter for 32 bit and they put system32 in the pipeProgram.